### ISSUE TITLE
PHP 8.2.1 Support Fails?

### ISSUE THREAD

**Issue by lautiamkok** [laminas/laminas-stratigility#36]
### Bug Report

I am trying to install the latest version 3.9 and I thought it supports PHP 8.2 already. But I have the following error during the installation:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laminas/laminas-escaper 2.10.0 requires php ^7.4 || ~8.0.0 || ~8.1.0 -> your php version (8.2.1) does not satisfy that requirement.
    - laminas/laminas-escaper[2.11.0, ..., 2.12.0] require ext-mbstring * -> it is missing from your system. Install or enable PHP's mbstring extension.
    - laminas/laminas-stratigility 3.9.0 requires laminas/laminas-escaper ^2.10.0 -> satisfiable by laminas/laminas-escaper[2.10.0, 2.11.0, 2.12.0].
    - Root composer.json requires laminas/laminas-stratigility ^3.9 -> satisfiable by laminas/laminas-stratigility[3.9.0].

To enable extensions, verify that they are enabled in your .ini files:
    - /etc/php/8.2/cli/php.ini
    - /etc/php/8.2/cli/conf.d/10-opcache.ini
    - /etc/php/8.2/cli/conf.d/10-pdo.ini
    - /etc/php/8.2/cli/conf.d/20-calendar.ini
    - /etc/php/8.2/cli/conf.d/20-ctype.ini
    - /etc/php/8.2/cli/conf.d/20-exif.ini
    - /etc/php/8.2/cli/conf.d/20-ffi.ini
    - /etc/php/8.2/cli/conf.d/20-fileinfo.ini
    - /etc/php/8.2/cli/conf.d/20-ftp.ini
    - /etc/php/8.2/cli/conf.d/20-gettext.ini
    - /etc/php/8.2/cli/conf.d/20-iconv.ini
    - /etc/php/8.2/cli/conf.d/20-imagick.ini
    - /etc/php/8.2/cli/conf.d/20-phar.ini
    - /etc/php/8.2/cli/conf.d/20-posix.ini
    - /etc/php/8.2/cli/conf.d/20-readline.ini
    - /etc/php/8.2/cli/conf.d/20-shmop.ini
    - /etc/php/8.2/cli/conf.d/20-sockets.ini
    - /etc/php/8.2/cli/conf.d/20-sysvmsg.ini
    - /etc/php/8.2/cli/conf.d/20-sysvsem.ini
    - /etc/php/8.2/cli/conf.d/20-sysvshm.ini
    - /etc/php/8.2/cli/conf.d/20-tokenizer.ini
You can also run `php --ini` inside terminal to see which files are used by PHP in CLI mode.
```

I have `mbstring` enabled already in my `php.ini`:

`extension=mbstring`

Thanks.

**Comment by boesing** [comment]
It seems that you do also require `laminas-escaper`. 
Try to run `composer update laminas/laminas-escaper` before you require `laminas/laminas-stratigility`.

if it does not let you install stratigility, you might want to check why `laminas/laminas-escaper` does not let you update to 2.11.0+ (which introduced PHP 8.2 support).

You can do that by running:

```
composer why-not laminas/laminas-escaper 2.11.0
```

----

Or a more simple alternative which might also lead to success:

```
composer require -W laminas/laminas-stratigility
```

This will update all packages which also might block the installation of stratigility. However, it seems that it is related to the escaper component and thus you need to update that.

Closing here as this component is compatible with PHP 8.2

**Comment by lautiamkok** [comment]
Thanks, @boesing . I have fixed the problem after reinstalling my PHP 8.2
